src/EmbeddedSonarAnalyzer.props

```diff
@@ -7,17 +7,4 @@
     <EmbeddedSonarCFamilyAnalyzerVersion>6.17.0.27551</EmbeddedSonarCFamilyAnalyzerVersion>
   </PropertyGroup>
 
-  <ItemGroup Label="Embedded SonarLint analyzers">
```

Comment:
Previously this props file was only used by Integration.Vsix and it's tests.

Going forward, a property describing the version of the SonarJS analyzer to embed will be added, so the props file will be referenced by the new additional files VSIX.

This change moves the Integration.Vsix-specific bits out of this file.